.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: CI
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DJANGO_SETTINGS_MODULE: rca.settings.test
+      SECRET_KEY: testingsecretkey
+      SECURE_SSL_REDIRECT: false
+      ALLOWED_HOSTS: localhost
+      DATABASE_URL: postgres://postgres:postgres@localhost/postgres
+      BIRDBATH_REQUIRED: false
+
+    services:
+      postgres:
+        image: postgres:11.5
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install npm dependencies
+        run: npm install
+      - name: CSS linting
+        run: npm run lint:css
+      - name: JS linting
+        run: npm run lint:js
+      - name: Prettier
+        run: npm run format
+      - name: JS tests
+        run: npm run test
+      - name: Compile static files
+        run: npm run build:prod
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - uses: Gr1N/setup-poetry@v8
+        with:
+          poetry-version: 1.5.0
+      - name: Install Poetry
+        run: |
+          poetry config virtualenvs.create false &&
+          poetry install
+      # - name: Flake8
+      #   run: flake8 ./rca fabfile.py
+      - name: isort
+        run: isort --check-only --diff ./rca fabfile.py
+      - name: black
+        run: black --check --diff ./rca fabfile.py --exclude migrations/
+      - name: collectstatic
+        run: python manage.py collectstatic --verbosity 0 --noinput --clear
```

Comment:
Yea we do need collectstatic, looks like you got it in her though :)

Reply:
If I do manage to split them out would `black` and `flake8` count as `lint` (along with the npm tasks) or `test` in your view?

Reply:
I have now split them out - but it repeats the set up for python , poetry and npm across both which is maybe adding more time than is ideal.